django_cyverse_auth/models.py

```diff
@@ -224,7 +223,7 @@ def lookupSessionToken(request):
     """
```

Comment:
OK, so the reason why i purposed this is,  in the upstream atmosphere code base, they use core_identity, identity, CoreIdentity to refer to one object, which is confusing. And there were some bugs that I helped fix. So, I am trying to avoid this in our code base. And I think it is reasonable to ask for changing the docs string along with the function name change. If both of you think it is unnecessary, it is fine with me. I will purpose a change by myself later.